### PR TITLE
Add sign-out link to header

### PR DESCRIPTION
**Pull request by harley-harris**
### JIRA link

[NTRNL-511](https://technologyprogramme.atlassian.net/jira/software/projects/NTRNL/boards/312?selectedIssue=NTRNL-511)

### Description

- Update CDN to new version to use GOV.UK styles required in sign out link
- Add sign out link to header

### Work checklist

- [x] Tests added where applicable
- [x] No vulnerability added

Screenshot 2024-07-09 at 12 51 01

[NTRNL-511]: https://technologyprogramme.atlassian.net/browse/NTRNL-511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Mouhajer-CO — COMMENTED** [review]
Hi @harley-harris, I would have added a separate component in the include folder and imported it in the layout, maintaining the separation between view and logic

**Review by Mouhajer-CO — COMMENTED** [review]
Awesome work 👍. I would suggest to add the `signout` block after the banner on the `layout.html` file and instead of duplicating to all pages to use the `userEmail` as condition on sign-out.html file.

File: src/views/layout.html [hunk 1]
```diff
@@ -86,6 +87,6 @@
   {% if FEATURE_FLAG_ENABLE_COOKIE_BANNER == "true" %}
     <script src="/assets/javascripts/cookies.js"></script>
   {% endif %}
-  <script src="//{{CDN_HOST}}/javascripts/govuk-frontend/v3.11.0/govuk-frontend-3.11.0.min.js"></script>
+  <script src="//{{CDN_HOST}}/javascripts/govuk-frontend/v5.3.0/govuk-frontend-5.3.0.min.js"></script>
```
Comment (Mouhajer-CO) [inline src/views/layout.html:90]: Based on new origin `/javascripts` should be removed from the path.

File: src/views/include/sign-out.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% if userEmail %}
+  <ul class="govuk-list" style="float: right;">
+    <li class="govuk-list govuk-!-margin-top-2">
+      <p>
```
Comment (Mouhajer-CO) [inline src/views/include/sign-out.html:4]: add `class="govuk-body"` to the `<p>` tag

**Review by Mouhajer-CO — COMMENTED** [review]
Thanks @harley-harris, I have created this story [here](https://technologyprogramme.atlassian.net/browse/NTRNL-515) to update the  `govuk-frontend` dependency to match our CDN version ^5.3.0

File: src/views/layout.html (hunk 1 above)
Comment (harley-harris) [reply src/views/layout.html:90]: Cool, I updated the CDN as the 3.11.0 didn't have some of the styles I needed for this component. Thanks for making the new ticket 👍

**Review by Mouhajer-CO — APPROVED** [review]
🔥

**Pull request merged**